### PR TITLE
Use the new `unused_images.pl` script to find such, and remove them.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/461700446828881
https://app.asana.com/0/353548986073982/461700446828877

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -60,6 +60,10 @@ spell:
 length:
 	@node_modules/gitbook-plugin-buddybuild/scripts/linelength.pl -d . -l 80
 
+# Check for any unused images
+unusedimg:
+	@node_modules/gitbook-plugin-buddybuild/scripts/unused_images.pl -d .
```
Comment (clintam) [inline Makefile]: did this file get added in another change?

**Comment by clintam** [comment]
love the idea. We have so much crap in BB.git we need to cleanup...

File: Makefile (hunk 1 above)
Comment (bbewan) [reply Makefile]: Yes. Script is here: https://github.com/buddybuild-public/gitbook-plugin-buddybuild/blob/master/scripts/unused_images.pl

**Comment by bbdocs** [comment]
See the website here: http://96-bb-docs.surge.sh

**Pull request merged**